build-files/Dockerfile

```diff
@@ -9,8 +9,10 @@ RUN apk update && apk upgrade && \
 RUN apk add --no-cache \
             jq \
             php7-session@community \
+            php7-pdo@community \
```

Comment:
Ha, yes. I've been feeling my way through it a lot. I probably need to read a LOT more documentation too. Just not enough time!